examples/nextjs-next-with-typescript/src/ProTip.tsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
 import Link from '@material-ui/core/Link';
-import SvgIcon from '@material-ui/core/SvgIcon';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
 import Typography from '@material-ui/core/Typography';
 
-function LightBulbIcon(props: any) {
+function LightBulbIcon(props: SvgIconProps) {
   return (
     <SvgIcon {...props}>
       <path d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z" />
     </SvgIcon>
   );
 }
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    margin: theme.spacing(6, 0, 3),
-  },
-  lightBulb: {
-    verticalAlign: 'middle',
-    marginRight: theme.spacing(1),
-  },
-}));
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
Could you rebase this on `next`. This change shouldn't be required anymore thanks to #15549.

Reply:
I'm not sure to follow. This pull request reduces the "ways" of doing thing in the codebase. Should #15549 be a global migration effort ?

Reply:
Oh this is still some extra file. I don't understand what this "ProTip" does for the demo. If there's something to learn here it should be part of the docs (either code with a TS example or a separate section in guides/typescript

Reply:
The ProTip is a component that guides people to the next step. Once they have a working frontend infrastructure, they can start building their layout. It's only here to link https://next.material-ui.com/getting-started/page-layout-examples/.

Reply:
Oh it is linked from the App. I was under the impression this was an orphaned file.

Reply:
What's the long term strategy? Should we remove Theme from all the demos and only mention it in the TypeScript guide? Do you want me to start this effort in this pull request (revert the changes)

Reply:
I can handle it.

Reply:
I have tried to revert but it doesn't work as we rely on npm in this demo.